data/migrations/20181012121743_fix_actions_table.js

```diff
@@ -0,0 +1,20 @@
+exports.up = function(knex, Promise) {
+  return knex.schema.createTable("actions", function(actions) {
+    actions.increments();
+
+    actions
+      .integer("project_id")
+      .unsigned()
+      .notNullable()
+      .references("id")
+      .inTable("projects");
+
+    actions.string("description", 128).notNullable();
+    actions.text("notes").notNullable();
+    actions.boolean("completed").defaultTo(false);
```

Comment:
Note that `defaultTo` isn't supported by sqlite3, but it's good to have in case you switch to a different database engine.